src/tools/run-make-support/src/lib.rs

```diff
@@ -117,13 +117,13 @@ pub fn set_host_rpath(cmd: &mut Command) {
     });
 }
 
-/// Check that all files in `dir1` exist and have the same content in `dir2`. Panic otherwise.
-pub fn recursive_diff(dir1: impl AsRef<Path>, dir2: impl AsRef<Path>) {
+/// Assert that all files in `dir1` exist and have the same content in `dir2`
+pub fn assert_recursive_eq(dir1: impl AsRef<Path>, dir2: impl AsRef<Path>) {
```

Comment:
This name does not make it immediately clear that we're talking about directories. Maybe `assert_dirs_are_equal`?

Reply:
Renamed to `assert_dirs_are_equal` and updated 1 test.